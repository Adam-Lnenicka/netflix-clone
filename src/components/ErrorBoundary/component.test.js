import ErrorBoundaryComponent from "./component";
import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("ErrorComponent ", () => {
  it("Match Snapshot", () => {
    const props = {
      errorName: "errorName",
      componentStack: "componentStack",
    };
    const wrapper = shallow(<ErrorBoundaryComponent {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
