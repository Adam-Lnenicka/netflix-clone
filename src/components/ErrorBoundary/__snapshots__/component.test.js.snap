// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorComponent  Match Snapshot 1`] = `
<div
  className="error-boundary"
>
  <h2>
    Sorry, something went wrong!
  </h2>
  <h3>
    Here is some info:
  </h3>
  <div>
    <p>
      Error Type:
    </p>
    <span>
      errorName
    </span>
  </div>
  <article>
    <p>
      Stacktrace:
    </p>
    <p
      key="0"
    >
      <span>
        componentStack
      </span>
    </p>
  </article>
</div>
`;
